modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -358,6 +391,25 @@ in
       );
       default = [ ];
     };
+
+    githubOwner = lib.mkOption {
+      type = lib.types.str;
+      description = ''
+        Github owner account of the bug reporter issue
+      '';
+    };
+    githubRepo = lib.mkOption {
+      type = lib.types.str;
+      description = ''
+        Github repo of the bug reporter issue
+      '';
+    };
+    githubToken = lib.mkOption {
+      type = lib.types.str;
+      description = ''
+        Personal token of the bug reporter Github account
+      '';
+    };
```

Comment:
Would it make sense to create a separate module for the control panel in the modules/common/services, move github stuff there as well as any other configuration required for it and simply enable it in the guivm? This way we can simplify guivm configuration and make it easier to maintain and reuse the code.

Reply:
Separate module is created due to the review.